cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -757,15 +758,18 @@ def _deserialize_gate_op(
             op = gate(*qubits)
         elif which_gate_type == 'noisechannel':
             which_channel_type = operation_proto.noisechannel.WhichOneof('channel_value')
+            print('dingdong')
             if which_channel_type == 'depolarizingchannel':
                 p = arg_func_langs.float_arg_from_proto(
                     operation_proto.noisechannel.depolarizingchannel.probability
                 )
+                num_qubits = operation_proto.noisechannel.depolarizingchannel.num_qubits
+                print(num_qubits)
```

Comment:
Should we check for `num_qubits == 0` here or is it OK to let it cause zero division in the DepolarizingChannel?

Reply:
Added.